evap/evaluation/models.py

```diff
@@ -1807,6 +1807,10 @@ def send_to_users_in_evaluations(self, evaluations, recipient_groups, use_cc, re
 
         for user, user_evaluations in user_evaluation_map.items():
             subject_params = {}
+            evaluations_with_date = dict()
+            for evaluation in user_evaluations:
+                evaluations_with_date[evaluation] = (evaluation.vote_end_date - date.today()).days
+            evaluations_with_date = sorted(evaluations_with_date.items(), key=operator.itemgetter(0))
```

Comment:
it looks like you are doing nothing with `evaluations_with_date`. maybe you forgot to hand these over in the `body_params`?